projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -264,6 +264,12 @@ function fetch_newsletter_categories() {
  * @return array
  */
 function get_newsletter_categories() {
+	// opt out of rendering when we display newsletter catgeories in the subscribe modal
+	$should_render_newsletter_categories = apply_filters( 'wpcom_newsletter_categories_location', 'block' ) === 'block';
+	if ( ! $should_render_newsletter_categories ) {
+		return array();
+	}
```

Comment:
Could you add a docblock here, so our codex can be automatically updated to include this new hook?

I would also recommend using a `jetpack` prefix if you can, if this is distributed in Jetpack.

Something like this, maybe?

```suggestion
	/**
	 * Overwrite the default Newsletter categories location.
	 *
	 * @module subscriptions
	 *
	 * @since $$next-version$$
	 *
	 * @param string $location The default location for the Newsletter categories. default to 'block'.
	 */
	$newsletter_categories_location = apply_filters( 'jetpack_newsletter_categories_location', 'block' );

	// Opt out of rendering when we display newsletter catgeories in the subscribe modal.
	if ( 'block' !== $newsletter_categories_location ) {
		return array();
	}
```

Reply:
Hey @jeherve . Sorry for getting back to this so late, I missed the notification.

This is only going to be here temporary. The newsletter categories will be moving from the block to the subscribe modal. The filter is just here to make the transition smooth (so we can flip the switch - and the categories should be gone everywhere). 

Therefor, I don't think this should be documented.

(Newsletter categories are only enabled on Simple & Atomic for now)

Reply:
Ok, that makes sense. If this ever comes up again, do not hesitate to note directly in the code when something is only there temporarily; this way we remember to take it off afterwards :)